metabase/metabase
# Migrate `list.module.css`

Epic https://github.com/metabase/metabase/issues/38811
Sub-task https://github.com/metabase/metabase/issues/41019

### Description

Migrate classes in `list.module.css`

### How to verify
TBD, but if you want to review in the mean time, please try to see where these classes are used. I'll fill more information later.

### Demo
TBD

### Checklist

- [] ~Tests have been added/updated to cover changes in this PR~ Only CSS has changed
